### PR TITLE
[complex.numbers] Use \xref for references to the C standard.

### LINKED ISSUE #1822: [complex.value.ops] and [complex.transcendentals] contain failed attempts to xref the C standard
Example:

screenshot from 2017-11-17 18 38 51

Err, no, that's not how we write cross-references to C.

Also, due to the nonstandard formatting, these were missed when rebasing on C11. The first xref (to `cproj`) should refer to 7.3.9.5 not 7.3.9.4. (The rest, by sheer luck, still seem to be OK, but someone should double-check.)

### PR DESCRIPTION
**Pull request by jensmaurer**
Fixes #1822.

### CODE REVIEW

File: source/numerics.tex [hunk 1]
```diff
@@ -995,7 +995,8 @@
 
 \pnum
 \remarks
-Behaves the same as the C function \tcode{cproj}, defined in 7.3.9.4.
+Behaves the same as the C function \tcode{cproj}.
+\xref ISO C 7.3.9.5
```
[the remark this answers is not in the document]
Comment (jensmaurer) [reply source/numerics.tex]: See my comment in #1822.

[the remark this answers is not in the document]
Comment (jensmaurer) [reply source/numerics.tex]: Can you ping Richard for an opinion?

[the remark this answers is not in the document]
Comment (tkoeppe) [reply source/numerics.tex]: @zygoloid: Ping :-)

[the remark this answers is not in the document]
Comment (zygoloid) [reply source/numerics.tex]: Let's go with this for now. It's clearly an improvement. But we should keep #1822 open to consider the consistency question.

**Pull request merged**